### PR TITLE
Remove most CHERI-MIPS support

### PR DESCRIPTION
**Pull request by arichardson**

### CODE REVIEW

File: githooks/pre-push.sh [hunk 1]
```diff
@@ -100,14 +100,14 @@ do
...
 		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --allow-more-than-one-target --build --test --cpu=default -p __run_everything__
 		# Regression for --benchmark-clean-boot:
-		try_run ./cheribuild.py mibench-mips64 --benchmark --benchmark-clean-boot -p
-		try_run ./cheribuild.py mibench-mips64-purecap --benchmark --benchmark-clean-boot -p
+		# TODO: try_run ./cheribuild.py mibench-new-riscv64 --benchmark --benchmark-clean-boot -p
+		# TODO: try_run ./cheribuild.py mibench-new-riscv64-purecap --benchmark --benchmark-clean-boot -p
 		# Various jenkins things that have failed in the past
 		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --build --test --tarball -p libcxx-riscv64-purecap
-		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --build --cpu=cheri128 -p run-mips64-purecap
+		try_run env WORKSPACE=/tmp ./jenkins-cheri-build.py --build --cpu=cheri128 -p run-morello-purecap
```
Comment (jrtc27) [inline githooks/pre-push.sh:107]: --cpu doesn't really make sense any more

File: pycheribuild/projects/syzkaller.py [hunk 2]
```diff
@@ -149,6 +150,7 @@ def clean(self) -> ThreadJoiner:
 
 class RunSyzkaller(SimpleProject):
     target = "run-syzkaller"
+    supported_architectures = [CompilationTargets.CHERIBSD_RISCV_HYBRID_FOR_PURECAP_ROOTFS]
```
[the remark this answers is not in the document]
Comment (arichardson) [reply pycheribuild/projects/syzkaller.py:153]: Yeah I could use that instead.

File: githooks/pre-push.sh (hunk 1 above)
Comment (arichardson) [reply githooks/pre-push.sh:107]: I agree, and it hasn't for a long time. Planning to remove that in a separate change though

File: README.md [hunk 3]
```diff
@@ -416,6 +416,18 @@ usage: cheribuild.py [-h] [--config-file FILE] [--help-all] [--pretend] [--build
                      [--cheribsd-mfs-root-kernel/build-alternate-abi-kernels | --cheribsd-mfs-root-kernel/no-build-alternate-abi-kernels]
                      [--cheribsd-mfs-root-kernel/build-bench-kernels | --cheribsd-mfs-root-kernel/no-build-bench-kernels]
                      [--cheribsd-mfs-root-kernel/caprevoke-kernel | --cheribsd-mfs-root-kernel/no-caprevoke-kernel]
+                     [--freebsd-release/build-tests | --freebsd-release/no-build-tests]
```
[the remark this answers is not in the document]
Comment (arichardson) [reply README.md:419]: If you feel like it I can rebase this PR afterwards but I don't think it really matters.

**Pull request merged**